restli-int-test/src/test/java/com/linkedin/restli/examples/TestGreetingUnstructuredDataResources.java

```diff
@@ -16,17 +16,15 @@
 
 package com.linkedin.restli.examples;
 
-
 import com.linkedin.restli.common.RestConstants;
-
+import java.io.BufferedReader;
+import java.io.InputStreamReader;
 import org.testng.annotations.DataProvider;
 import org.testng.annotations.Test;
 
-import static com.linkedin.restli.examples.greetings.server.GreetingUnstructuredDataUtils.UNSTRUCTURED_DATA_BYTES;
-import static com.linkedin.restli.common.RestConstants.HEADER_CONTENT_DISPOSITION;
-import static com.linkedin.restli.examples.greetings.server.GreetingUnstructuredDataUtils.CONTENT_DISPOSITION_VALUE;
-import static com.linkedin.restli.examples.greetings.server.GreetingUnstructuredDataUtils.MIME_TYPE;
-import static org.testng.Assert.assertEquals;
+import static com.linkedin.restli.common.RestConstants.*;
+import static com.linkedin.restli.examples.greetings.server.GreetingUnstructuredDataUtils.*;
+import static org.testng.Assert.*;
```

Comment:
nit: We should import particular classes rather than using *.

Reply:
noticed